dev-ops/docker/containers/redis/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM redis:4
```

Comment:
This file is not nessessary, you can directly reference the image in your docker-compose file.